Shopping.java

```diff
@@ -0,0 +1,190 @@
+	public static String continueBtnChkOutLocator3="xpath=.//*[@id='shipping-method-buttons-container']/button";
+	public static String paymentMoneyRbtLocator="xpath=.//*[@id='checkout-payment-method-load']/dt[1]/label";
+	public static String continueBtnChkOutLocator4="xpath=.//*[@id='payment-buttons-container']/button";
+	public static String placeOrderBtnLocator = "xpath=.//*[@id='review-buttons-container']/button";
+	public static String pageTitleLocator = "xpath=.//*[@class='page-title']";
+	
+	// Choose item
+	public static void chooseItem(String locator, String item) {
+		Select select = new Select(driver.findElement(By.xpath(locator)));
+		select.selectByVisibleText(item);
+	}
+
+	//Input info sau đó click add to cart
+	public static void addToCart() throws InterruptedException {
+		CommonBase common = new CommonBase(driver);
+		try {
+			// Chọn mua tivi
+			common.click(tvMenuLocator);
+			Thread.sleep(3000);
+
+			// Chọn sản phẩm
+			common.click(productLocator);
+			Thread.sleep(3000);
+
+			// Nhập số lượng muốn mua
+			common.clear(quantityLocator);
+			common.sendKeys(quantityLocator, soLuong);
+
+			// click add to cart
+			common.click(addToCartLocator);
+		} catch (Exception e) {
+			common.printLn(e.toString());
+		}
+	}
+
+	//Input info sau đó click proceed to check Out
+	public static void proceedToCheckOut() throws InterruptedException {
+		CommonBase common = new CommonBase(driver);
+		try {
+			// Chọn country
+			chooseItem(countryLocator, country);
+
+			// Chọn province
+			common.sendKeys(provinceLocator, province);
+
+			// click proceed to checkout
+			common.click(processBtnLocator);
+			Thread.sleep(3000);
+		} catch (Exception e) {
+			common.printLn(e.toString());
+		}
+	}
+
+	//Input billing info sau đó click CheckOut
+	public static void billingInforCheckOut() throws InterruptedException {
+		CommonBase common = new CommonBase(driver);
+		try {
+			// Choose checkout as Guest
+			common.click(checkOutRbtLocator);
+
+			// click continue
+			common.click(continueBtnChkOutLocator);
+			Thread.sleep(3000);
+
+			// input info
+			common.sendKeys(blFristNameLocator, blFristName);
+			common.sendKeys(blLastNameLocator, blLastName);
+			common.sendKeys(blCompanyLocator, blCompany);
+			common.sendKeys(blEmailLocator, blEmail);
+			common.sendKeys(blAddressLocator, blAddress);
+			common.sendKeys(blStAddressLocator, blStAddress);
+			common.sendKeys(blCityLocator, blCity);
+			chooseItem(blProvinceLocator, blProvince);
+			common.sendKeys(blZipLocator, blZip);
+			chooseItem(blCountryLocator, blCountry);
+			common.sendKeys(blTelLocator, blTel);
+			common.sendKeys(blFaxLocator, blFax);
+			
+			// click continue
+			common.click(continueBtnChkOutLocator2);
+			Thread.sleep(5000);
+			
+			// click continue next
+			common.click(continueBtnChkOutLocator3);
+			Thread.sleep(3000);
+			
+			// chọn thanh toán tiền mặt
+			common.click(paymentMoneyRbtLocator);
```

Comment:
1. The same comments with above functions?
2. Should split function payment method